lib/util/stream_utils.js

```diff
@@ -128,7 +128,7 @@ shaka.util.StreamUtils = class {
     const validVideoIds = [];
     const validVideoStreamsMap = new Map();
     const getVideoGroupId = (stream) => {
-      return (stream.frameRate || 0) + (stream.hdr || '');
+      return Math.round(stream.frameRate || 0) + (stream.hdr || '');
```

Comment:
Wouldn't this result in NTSC 29.97 and 30 being treated as the same FPS?

Reply:
Yes! But I think that nobody uses both in the same stream